Hw1/Task4.py

```diff
@@ -0,0 +1,29 @@
+# Задача 4: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, 
+# если разрешается сделать один разлом по прямой между дольками 
+# (то есть разломить шоколадку на два прямоугольника).
+
+size1 = input("Введите длину шоколадки: ")
+
+n = int(size1) 
```

Comment:
А почему бы это не совместить?
```suggestion
n = int(input("Введите длину шоколадки: "))
```